01-Login/views/assets/style.css

```diff
@@ -0,0 +1,22 @@
+.mainHeader{
+  padding: 20px 30px;
+  display: flex;
+  align-items: center;
+  background-color: #27417E;
+  color: white;
+}
+.header-text {
```

Comment:
I feel like this font dooesnt really look like a header. can we make it bigger? i know this is nitpicky